_data/hosting_providers.json

```
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "WebOké",
+    "link": "https://weboke.nl/",
+    "category": "full",
```

```suggestion
    "category": "partial",
```

Since the users have to take steps to turn on the certificate, that's what we call "partial" support.